<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 216, 89)</string>
  </property>
  <widget class="QLabel" name="med_input_header">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AccessRx</string>
   </property>
  </widget>
  <widget class="QLabel" name="med_input_subheader">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Prototype Gui For Pitt Challenge 2022</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>We weren't able to find your medication from the image.</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="med_text_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; color: black; border: 1px solid black;</string>
    </property>
    <property name="plainText">
     <string>Medication Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="med_input_send">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 174, 0); border: 1px solid black; border-radius: 5px;</string>
    </property>
    <property name="text">
     <string>Done!</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Please input the information : </string>
    </property>
   </widget>
   <widget class="QComboBox" name="med_dosage_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border: 1px solid black; color: black;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="med_freq_input">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white; border: 1px solid black; color: black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="med_dosage_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Medication dosage : </string>
    </property>
   </widget>
   <widget class="QLabel" name="med_freq_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Medication frequency : </string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
